### PR TITLE
Tweak nav styling

### PR DESCRIPTION
**Pull request by Feeshay**
## Status:
:rocket: Ready

## Description :star2:
* Make nav bars align closer to Faith's designs
  * Make pages extend to full height of window
  * Fix menu item spacing to get rid of this annoying gray bar on the right side of the sidebar (this took me so long to figure out jfc)
  * Add shadow to sidebar
* Add profile CSS file

Before:
image
After:
image

### CODE REVIEW

**Review by Leundai — APPROVED** [review]
Looks good!

File: frontend/src/components/css/Navigation.scss [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 }
 
 .navigation-header {
   background-color: #fbecc1;
   padding: 0px;
 }
 
+.sidebar {
+    box-shadow: 0px 2px 16px rgba(0, 0, 0, 0.244974);
+    border-radius: 0px 8px 8px 0px;
```
Comment (AngelaLuo49021) [inline frontend/src/components/css/Navigation.scss:13]: Remove the rounded borders

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
Looks gucci overall! Just remove that one rounded border I just talked to faith about and merge in lam's changes from master

**Review by AngelaLuo49021 — APPROVED** [review]
very purty!

**Pull request merged**